src/VodafoneClient.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace NotificationChannels\Vodafone;
+
+use GuzzleHttp\Client;
+use NotificationChannels\Vodafone\Exceptions\CouldNotSendNotification;
+
+class VodafoneClient
+{
+    /**
+     * @var string Vodafone's API endpoint
+     */
+    protected $endpoint = 'https://www.smsertech.com/apisend';
+
+    /** @var string Vodafone SMS username */
+    protected $username;
+
+    /** @var string Vodafone SMS password */
+    protected $password;
+
+    /**
+     * VodafoneClient constructor.
+     * @param $username
+     * @param $password
+     */
+    public function __construct()
+    {
+        $this->username = config('services.vodafone.username');
+        $this->password = config('services.vodafone.password');
+
+        return $this;
+    }
+
+    /**
+     * @param $from
+     * @param $to
+     * @param $message
+     * @return mixed Vodafone API result
+     */
+    public function send($from, $to, $message)
+    {
+        $client = new Client();
+        $res = $client->post($this->endpoint, [
+            'form_params' => [
+                'username' => $this->username,
+                'password' => $this->password,
+                'to' => $to,
+                'message' => $message,
+                'from' => $from,
+                'format' => 'json',
+                'flash' => 0,
+            ],
+        ]);
+
+        if (! $res) {
+            throw CouldNotSendNotification::serviceUnknownResponse();
+        }
+
+        $body = json_decode($res->getBody()->getContents())[0];
```

Comment:
```suggestion
        $body = json_decode($res->getBody()->getContents())[0] ?? null;
```

Might be worth defaulting to null here